tests/format/yaml/spec/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -3814,8 +3814,7 @@ proseWrap: "always"
 =====================================output=====================================
 %TAG ! tag:clarkevans.com,2002:
 ---
-!shape
-# Use the ! handle for presenting
+!shape # Use the ! handle for presenting
```

Comment:
These changes shouldn't happen. Comments that are on their own line should stay on their line, trailing comments should stay trailing.